package.json

```diff
@@ -33,6 +34,7 @@
     "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
     "test:watch": "npm run test -- -w ./lib",
     "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
+    "prettier": "prettier --single-quote --trailing-comma es5 --write **/*.js *.js",
```

Comment:
The new npm target should have its corresponding entry under https://github.com/telefonicaid/iotagent-node-lib/tree/master/doc/usermanual.md#development-documentation

Reply:
Fixed 75c0c54

Reply:
Looking to prettier option, I'd suggest to included `--tab-width 4` to keep the current indentation style.

I'd like to start a passionate discussion on indentation length ;), but the fact is that this repository code uses 4 from its very begining, so let's keep that.

Reply:
Fixed a4979bd  + 998f379 - This should make the diff less messy.

On the whole I've gone with the defaults suggested by prettier themselves, but double/single quote is repo preference and trailing-comma es5 is a step to the future (it adds the trailing quote only where ecma5 accepts it

 Ideally once node 6 support is deprecated this can be updated to be consistent for ecma6 and use `let` and `const` and arrow functions and so on, but at least it is a start (and won't break the code)